doc/source/rest.j2

```diff
@@ -551,15 +551,85 @@ argument, and in this case the second argument passed is the value, or it.
 The operators or (`or` or `∨`), and (`and` or `∧`) and `not` are also
 supported, and take a list of arguments as parameters.
 
+.. _aggregates:
+
+Aggregates: on the fly, measurements modification and aggregatation
+===================================================================
+
+Gnocchi allows to do on-the-fly aggregation and modification of already
+aggregated data of |metrics|.
+
+It can be done by providing the list of |metrics| to aggregate:
+
+{{ scenarios['get-aggregates-by-metric-ids']['doc'] }}
+
+This example computes the mean aggregates with `all` metrics listed in
+`metrics` and then multiples it by `4`.
+
+.. Note:
+
+   `operations` can also be passed as a lisp string, for this example:
+   `"operations": "(aggregate mean (metric all))"`
+
+Operations between metrics can also be done, such as:
+
+{{ scenarios['get-aggregates-between-metrics']['doc'] }}
+
+Partial list of supported maths/aggregation::
```

Comment:
Can we have the full list with their signature please? :)

Reply:
It's plan to expose the list through the API, when it's done, I can fixes that comment :p

Reply:
Well, that'd be great but I'm not sure it will explain what the method really do with proper English, no? :(